selfdrive/controls/controlsd.py

```diff
@@ -77,6 +75,8 @@ def data_sample(CI, CC, sm, can_sock, driver_status, state, mismatch_counter, ca
   sm.update(0)
 
   events = list(CS.events)
+  if sm.updated['monitorState']:
```

Comment:
Don't you always want to add the events?

Reply:
fixed. experienced this bug on my test drive too